bindings/c/fdb_c.cpp

```diff
@@ -366,6 +366,11 @@ extern "C" DLLEXPORT FDBFuture* fdb_database_create_snapshot(FDBDatabase* db,
 	                        .extractPtr());
 }
 
+// Get network thread busyness (updated every 1s)
```

Comment:
Can you please describe the return value briefly? I imagine 0 means "did nothing" and "1.0" means "CPU is pegged"?